README.md

```diff
@@ -55,8 +64,8 @@ hipblasSgemv( hipblasHandle_t handle,
 
 ### Batched and strided GEMM API
 
-hipBLAS GEMM can process matrices in batches with regular strides.  There are several permutations of these API's, the
-following is an example that takes everything
+hipBLAS GEMM can process matrices in batches with regular strides. The following example uses all
+permutations of the API.
 
```

Comment:
This sentence (prior to and after changes) is confusing to me, I'm not really sure what it's even trying to say. Something like the following may be better:
"hipBLAS GEMM can process matrices in batches with regular strides by using the strided-batched version of the API as follows:"

Reply:
My interpretation was that the example included all possible options. But your edit allows room for future additions without content modification, so seems like a better option regardless.